internal/relayertest/testdata/default_config.toml

```diff
@@ -8,5 +8,6 @@ penalty_exponential_factor = 1.0
 [bandchain]
 rpc_endpoints = ['http://localhost:26657']
 timeout = 3000000000
+liveliness_checking_interval = 900000000000
```

Comment:
I think the liveliness internal should be quite fast at most (<=5m / <=1m) for both the band side and the chain side because we don't have the logic to switch nodes if it fails. the program will stuck until the next interval.

@tanut32039 @nkitlabs what do you think? 

anyway, please test on fast intervals as well. to make sure that we don't have an issue if the node switches in the middle of the process.

Reply:
the default interval can be reduced.